packages/mds-stream/index.ts

```diff
@@ -168,23 +193,23 @@ async function writeStreamBatch(stream: Stream, field: string, values: unknown[]
 
 // put basics of vehicle in the cache
 async function writeDevice(device: Device) {
-  if (env.SINK) {
-    return writeCloudEvent('device', JSON.stringify(device))
+  if (env.NATS) {
+    return writeNatsEvent('device', JSON.stringify(device))
```

Comment:
Should `writeCloudEvent` be deleted now? I don't see any calls to it.

Reply:
Added a comment explaining why we're keeping it still